genesis/model/resolver/global_test.go

```diff
@@ -15,3 +15,53 @@
  */
 
 package resolver
+
+import (
+	"testing"
+
+	"github.com/insolar/insolar/genesis/model/object"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewGlobalResolver(t *testing.T) {
+	resolver := newGlobalResolver()
+	instanceMap := make(map[*object.Reference]object.Proxy)
+
+	assert.Equal(t, &globalResolver{
+		globalInstanceMap: &instanceMap,
+	}, resolver)
+}
+
+func TestGlobalResolver_GetObject_No_Object(t *testing.T) {
+	resolver := newGlobalResolver()
+	ref, _ := object.NewReference("1", "1", object.GlobalScope)
+
+	obj, err := resolver.GetObject(ref, "someClass")
+
+	assert.Equal(t, "reference with address `#1.#1` not found", err.Error())
```

Comment:
You could use `assert.EqualError` for error checks.

`assert.EqualError(t, err, "reference with address `#1.#1` not found")`